src/Abp.Zero.EntityFramework/EntityHistory/EntityHistoryHelper.cs

```diff
@@ -487,6 +487,22 @@ private void UpdateChangeSet(DbContext context, EntityChangeSet changeSet)
                 var entityType = GetEntityType(context.As<IObjectContextAdapter>().ObjectContext, entityEntry.GetEntityBaseType(), useClrType: false);
                 entityChange.EntityId = GetEntityId(entityEntry, entityType);
 
+                /* Update audit field */
+
+                foreach (var propertyChange in entityChange.PropertyChanges.
+                    Where(x => x.PropertyName == nameof(IFullAudited.CreationTime) ||
+                               x.PropertyName == nameof(IFullAudited.CreatorUserId) ||
+                               x.PropertyName == nameof(IFullAudited.LastModificationTime) ||
+                               x.PropertyName == nameof(IFullAudited.LastModifierUserId)))
+                {
+                    var propertyEntry = entityEntry.Property(propertyChange.PropertyName);
+                    if (propertyEntry != null && propertyChange.NewValue != propertyEntry.CurrentValue.ToJsonString().TruncateWithPostfix(EntityPropertyChange.MaxValueLength))
+                    {
+                        propertyChange.NewValue = propertyEntry.CurrentValue.ToJsonString()
```

Comment:
for ef6, you need to use the extension here to safely access `NewValue` of `PropertyEntry`.

https://github.com/aspnetboilerplate/aspnetboilerplate/blob/d95f52a0e950ea51f5ddffb274ebedf52d6daef7/src/Abp.Zero.EntityFramework/EntityHistory/Extensions/DbPropertyEntryExtensions.cs#L8

Reply:
It seems that I am not using the `NewValue `property of `propertyEntry`.

Reply:
CurrentValue means NewValue
OriginalValue means OldValue